docs/setup/tomcat.md

```diff
@@ -73,59 +78,96 @@ echo "" | openssl s_client -connect LDAPHOST:LDAPPORT -showcerts 2>/dev/null | o
 sudo keytool -import -alias cert_ldap -file /tmp/certfile.txt -keystore /etc/tomcat9/keystore
 ```
 
-### Finally, 
+### Finally,
+
 verify the list of keys in keystore:
 ```
 keytool -keystore /etc/tomcat9/keystore -list
 ```
 
+## geOrchestra datadir
+
+Since the 15.12 version, geOrchestra allows its main configuration settings to be done into a specific directory, the "georchestra Datadir".
```

Comment:
More than "allows", it is now required.

Reply:
you're right, at the begining of the geor datadir, it was not mandatory though.